e2e Validation for Modeling Rule fields in Schema/TestData (#3290)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased

> * Added support for the **modeling-rules test** command to validate whether the modeling rules schema mappings are aligned with the test-data mappings.

Reviewer: ```suggestion
* The **modeling-rules test** command will now validate that the modeling rules schema mappings are aligned with the test-data mappings.
```

> * The **modeling-rules test** command will now validate that the modeling rules schema mappings are aligned with the test-data mappings.

Reviewer 2: Why not your addition in green?

Author: should be good
